README.rst

```diff
@@ -225,7 +184,7 @@ Check status of a task
 
 ``$ http GET http://localhost:8000/pulp/pulp/api/v3/tasks/82e64412-47f8-4dd4-aa55-9de89a6c549b/``
 
-Download ``$CONTENT_NAME`` from Pulp
-------------------------------------------------------------------
+Perform a docker pull from Pulp
+-------------------------------
 
-``$ http GET http://localhost:8000/pulp/content/foo/$CONTENT_NAME``
+``$ sudo docker pull localhost:8000/foo``
```

Comment:
s/sudo//

If set up correctly (IMO) docker pull should be sudoless. Other setups can require sudo, but IMO the user will know that already.